patch-cli.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: page
+title: Querying for a patch plan via CLI
+permalink: /patch-plan/
+parent: GUAC demos
+grand_parent: Getting started with GUAC
+nav_order: 3
+---
+
+# Query for a patch plan via CLI Demo
+
+In this demo, we will utilize a Go CLI that will allow us to input a vulnerable
+purl (package URL) and see what else is affected by this package being
+vulnerable-- meaning we are querying for the package's dependents. This tutorial
+assumes you know which package has some sort of vulnerability or malware
+affecting it, or which package is your concern.
+
+We will get back a patch plan which includes a visual representation of the
+blast radius, meaning a link to view the subgraph of dependent packages in the
+GUAC visualizer. It will also return a list of frontiers. These frontiers
+represent what can be patched to resolve the vulnerability, starting from the
+lowest level possible, i.e. the package itself.  
+If you don't have access to fix this package directly, you could look in the
+next frontier and so on. It will also list out and points of contact for a given
+node, if available in the graph.  
+This will help you know who to contact to resolve the issue. All these outputs
+together can be used to help figure out the best plan of action in order to
+resolve a security incident.
+
+## Step 1: Setup GUAC with Docker Compose
+
+Follow the [Docker Compose setup]({{ site.baseurl }}{%link setup.md %}).
+
+## Step 2: Clone GUAC
+
+1. Clone GUAC to a local directory:
+
+   ```bash
+   git clone https://github.com/guacsec/guac.git
+   ```
+
+2. Clone GUAC data (this is used as test data for this demo):
+
+   ```bash
+   git clone https://github.com/guacsec/guac-data.git
+   ```
+
+The rest of the demo will assume you are in the GUAC directory.
+
+```bash
+cd guac
+```
+
+## Step 3: Build the GUAC binaries
```

Comment:
I know @jeffmendoza is updating the docs so that we do not have to build the local binaries

Reply:
So should I remove this?

Reply:
I removed that step but let me know if that's what you meant.

Reply:
Keep it's for now. Seems like the changes have not been made yet.